docs/commands/New-MockObject.mdx

```diff
@@ -71,6 +75,10 @@ Call Arguments
    3 {False}
```

Comment:
@bravo-kernel It tried to trim leading whitespace in this table. Is that expected inside a codefence or is it a bug?

Reply:
I will need to investigate this one but understand why this could be happening. Does not block merging correct?

Reply:
No, added the missing whitespace before submitting the PR.
Only that and the `\<` + `\>` below that had to be fixed this time so almost seamless conversion now 👏